### PR TITLE
Ritesh/add meta tags

### PR DESCRIPTION
**Pull request by Ritesh-Sharma8988**
changes:
-updated the title
-uploaded images banner image and favicon image
-added meta tags including og tags and twitter tags
-added favicon 

test cases:
-title is showing

### CODE REVIEW

**Comment by sharmaritesh707** [comment]
test cases for:
-are meta tags working
-is favicon working

**Comment by Ritesh-Sharma8988** [comment]
test cases:
-meta tags are working
-favicon is also working

File: src/index.html [hunk 1]
```diff
@@ -1,263 +1,279 @@
...
-          <div class="big-text">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus
-            turpis ligula, pellentesque a elit in, convallis laoreet metus. Sed
-            aliquet lacinia dictum. In pharetra nunc nisi, ac aliquet est
-            pretium non.
-          </div>
-          <button>Contact Me</button>
-        </div>
+<head>
+  <title>Jovian Careers</title>
```
Comment (sharmaritesh707) [inline src/index.html]: the title should be more than 22 characters

File: src/index.html [hunk 2]
```diff
@@ -1,263 +1,279 @@
...
+  <meta name="robots" content="index, follow">
+  <meta property="og:title" content="Jovian Careers">
+  <meta property="og:description" content="Learn about job openings at Jovian and submit your application now">
+  <meta property="og:image" content="/hero.png">
+  <meta name="twitter:title" content="Jovian Careers">
+  <meta name="twitter:description" content="Learn about job openings at Jovian and submit your application now">
+  <meta name="twitter:image" content="/hero.png">
+  <link rel="icon" href="img1.png" type="image/x-icon">
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```
Comment (sharmaritesh707) [inline src/index.html:19]: check if meta tags are working properly

**Review by sharmaritesh707 — CHANGES_REQUESTED** [review]
Check for these

File: src/index.html (hunk 2 above)
Comment (Ritesh-Sharma8988) [reply src/index.html:19]: meta tags are working properly

**Review by sharmaritesh707 — APPROVED** [review]
everything fine...

**Comment by Ritesh-Sharma8988** [comment]
changed title , favicon and metatags

**Pull request merged**